executor/adapter.go

```diff
@@ -876,14 +876,38 @@ func (a *ExecStmt) SummaryStmt() {
 	stmtCtx := sessVars.StmtCtx
 	normalizedSQL, digest := stmtCtx.SQLDigest()
 	costTime := time.Since(sessVars.StartTime)
+
+	var tableIDs, indexNames string
+	if len(stmtCtx.TableIDs) > 0 {
+		tableIDs = strings.Replace(fmt.Sprintf("%v", stmtCtx.TableIDs), " ", ",", -1)
```

Comment:
I used to want to change `tableIDs` to  `tableNames` in slow log, because the `tableID` is not friendly to the user. But later some one think the `tableID/ tableName` was useless, so I doesn't change it and don't record `tableID/tableName` in slow log.

If you wanto record the table name/ID in statement, I suggest you record the table name, not the ID.

Reply:
@zz-jason @jackysp PTAL

Reply:
Yes, and yesterday PM also recommends that. I'll modify it.

Reply:
Addressed.